tangelo/algorithms/classical/tests/test_ccsd_solver.py

```diff
@@ -41,13 +42,22 @@ def test_ccsd_h4_uhf_a1_frozen(self):
 
         self.assertAlmostEqual(mol_H4_sto3g_uhf_a1_frozen.energy_from_rdms(one_rdms, two_rdms), -1.95831052, places=6)
 
+    def test_ccsd_h4_uhf_different_alpha_beta_frozen(self):
+        """Test energy for case when different but equal number of alpha/beta orbitals are frozen."""
+
+        mol = SecondQuantizedMolecule(xyz_H4, q=0, spin=0, basis="3-21g", frozen_orbitals=[[2, 3, 4, 5], [2, 3, 6, 5]], symmetry=False, uhf=True)
+        solver = CCSDSolver(mol)
+        energy = solver.simulate()
+
+        self.assertAlmostEqual(energy, -2.0409800, places=5)
+
     def test_ccsd_be(self):
         """Test CCSDSolver against result from reference implementation."""
 
         solver = CCSDSolver(mol_Be_321g)
         energy = solver.simulate()
 
-        self.assertAlmostEqual(energy, -14.531416589890926, places=6)
+        self.assertAlmostEqual(energy, -14.531416589890926, places=4)
```

Comment:
a bit odd seeing 10+ digits (e.g up to questionable machine-precision) while we only care for 5-6.

Can you briefly explain why we relaxed those tests a bit, here?
Is it simply related to the differences between how psi4 and pyscf go about the computation, which are both deemed "correct" ?

Reply:
They are both correct to well within chemical accuracy. I assume it may be from the SCF/CCSD convergence threshold or numerical precision of the integrals.